\documentclass[]{article}
\usepackage[T1]{fontenc}
\usepackage{lmodern}
\usepackage{amssymb,amsmath}
\usepackage{ifxetex,ifluatex}
\usepackage{fixltx2e} % provides \textsubscript
% use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
\ifnum 0\ifxetex 1\fi\ifluatex 1\fi=0 % if pdftex
  \usepackage[utf8]{inputenc}
\else % if luatex or xelatex
  \ifxetex
    \usepackage{mathspec}
    \usepackage{xltxtra,xunicode}
  \else
    \usepackage{fontspec}
  \fi
  \defaultfontfeatures{Mapping=tex-text,Scale=MatchLowercase}
  \newcommand{\euro}{â‚¬}
\fi
% use microtype if available
\IfFileExists{microtype.sty}{\usepackage{microtype}}{}
\ifxetex
  \usepackage[setpagesize=false, % page size defined by xetex
              unicode=false, % unicode breaks when used with xetex
              xetex]{hyperref}
\else
  \usepackage[unicode=true]{hyperref}
\fi
\hypersetup{breaklinks=true,
            bookmarks=true,
            pdfauthor={},
            pdftitle={},
            colorlinks=true,
            citecolor=blue,
            urlcolor=blue,
            linkcolor=magenta,
            pdfborder={0 0 0}}
\urlstyle{same}  % don't use monospace font for urls
\setlength{\parindent}{0pt}
\setlength{\parskip}{6pt plus 2pt minus 1pt}
\setlength{\emergencystretch}{3em}  % prevent overfull lines
\setcounter{secnumdepth}{0}

\author{}
\date{}

\begin{document}

\begin{center}
\section{Amit R. Indap}\label{amit-r.-indap}
indapa@gmail.com

 http://www.linkedin.com/in/aindap
\end{center}

\begin{center}\rule{3in}{0.4pt}\end{center}

\begin{quote}
\begin{center}
Specialties: Bioinformatics, Human Genetics, Python
\end{center}
\end{quote}

\begin{center}\rule{3in}{0.4pt}\end{center}

\subsection{Experience}\label{experience}

\begin{description}
\item[Since December 2013]
\emph{Research Fellow for
\href{http://www.massgeneral.org/reproductiveendocrine//}{Reproductive
Endocrine Unit, Massachusetts General Hospital}} Boston, Massachusetts

Investigating the genetic basis of reproductive endocrine disorders
by whole exome and genome sequencing. Applying bioinformatic and statistical genomics 
methods to prioritize potential causal alleles. Working in an interdisciplinary group of physicians, genetic counselors,
and basic scientists. 
\item[January 2009 - October 2013]
\emph{PhD in Bioinformatics with
\href{http://bioinformatics.bc.edu/marthlab/wiki/index.php/Main_Page}{Gabor
T. Marth}} Boston College Chestnut Hill, Massachusetts

Utilized existing and wrote novel software to analye data from
population scale and family based whole genome and whole exome
re-sequencing studies to characterize the contribution of rare variants
to human genetic diversity and disease risk. Projects included
analyzing capture sequencing data from the 1000 Genomes Pilot Project, identifying a potential
causal mutation in a Mendelian disease, demonstrating the applicability of whole-genome amplified DNA
in capture sequencing, and implementing a pedigree aware variant calling algorithm. 

\item[August 2005 - December 2008]
\emph{Bioinformatics Programmer \href{http://www.cornell.edu/}{Cornell
University}} Ithaca, New York

Implemented bioinformatic data analysis pipelines for DNA re-sequencing
and genotyping datasets for research projects in population and
evolutionary genomics which helped facilitate the publication of over 10
papers in leading scientific journals.
\item[February 2001 - July 2002]
\emph{Bioinformatics Programmer at
\href{http://www.stanford.edu}{Stanford University}} Stanford, CA

Implemented informatics pipeline for sequencing and genotyping of
candidate genes involved in complex diseases
\end{description}

\subsection{Selected Publications}\label{selected-publications}

\href{http://www.ncbi.nlm.nih.gov/pubmed/23837845}{Variant discovery in
targeted resequencing using whole genome amplified DNA}

\href{http://www.ncbi.nlm.nih.gov/pubmed/21917140}{The functional
spectrum of low-frequency coding variation}

\href{http://www.ncbi.nlm.nih.gov/pubmed/20981092}{A map of human genome
variation from population-scale sequencing}

\href{http://www.ncbi.nlm.nih.gov/pubmed/18516229}{Assessing the
evolutionary impact of amino acid mutations in the human genome}

\href{http://www.ncbi.nlm.nih.gov/pubmed/18288194}{Proportionally more
deleterious genetic variation in European than in African populations}

\href{http://www.ncbi.nlm.nih.gov/pubmed/16356172}{Analysis of
concordance of different haplotype block partitioning algorithms}

\subsection{Skills}\label{skills}

Bioinformatics : GATK, MOSAIK, Freebayes, PLINK, samtools, Picard

Programming : Python, R, Perl, familiar with Java and C++,
\href{https://github.com}{GitHub repo}

Molecular Biology : PCR, gel electrophoresis

\subsection{Education}\label{education}

\begin{description}
\item[2009-2013]
\emph{PhD in Bioinformatics at Boston College} Chestnut Hill ,
Massachusetts

Title: Discovering rare variants from populations to families. Advisor:
\href{http://bioinformatics.bc.edu/marthlab/wiki/index.php/Main_Page}{Gabor
T. Marth} Defended in October 2013
\item[2003-2005]
\emph{MS in Bioinformatics at Medical College of Wisconsin and Marquette
University} Milwaukee, Wisconsin

Title: Analysis of concordance of haplotype block partitioning
algorithms. Advisor: Michael Olivier
\item[1996-2000]
\emph{BS in Molecular and Cellular Biology, minor Computer Science
University of Arizona} Tucson, Arizona

Selected as Outstanding Graduating Senior in Department of Molecular and
Cellular Biology
\end{description}





\end{document}
