Amit R. Indap
=============

indapa@gmail.com | [LinkedIn](http://www.linkedin.com/in/aindap) 

* * * * *

> Specialties: Human Genetics,Bioinformatics, Immunogenomics 
* * * * *

Experience
----------

## Senior Scientist
Pfizer San Diego, California
September 2016 - current

- Responsible for bioinformatics analyses in support of cancer and prophylactic vaccine programs

- Implemented diverse repoitoire of analytical pipelines for long-read denovo assembly, RNA-seq, epitope prediction

-Awarded Upjohn Prize in recognition of going above and beyond expected job description responsibilities.

## Research Scientist 
Human Longevity Inc.  San Diego, California
September 2015 - September 2016

- Implementing variant reporting pipelines for pharmacogenetics and autoimmune 
        diseases for HLI whole genome and exome reports using Docker and AWS
    
- Investigating distribution and frequency of loss-of-function variants in HLI cohort of samples


## Research Scientist 
Good Start Genetics Cambridge, MA
July 2014 - August 2015

 -  Performed bioinformatic and population genetic analyses on
        Ilumina sequencing data sets on tens of thousands of samples

-   Evaluated and recommend software for improvements of
    bioinformatics pipeline using latest methods state-of-the-art
    methods for variant annotation

-   Provide expertise in bioinformatic interpretation of reportable
    variants for clinical laboratory directors

##Research Fellow 
Massachusetts General Hospital Boston, MA
December 2013 - July 2014
   
-   Analyzed whole exome sequencing datasets to investigate the
    genetic basis of reproductive endocrine disorders

-   Applied bioinformatic and statistical genomics methods to
    prioritize potential pathogenic alleles

-   Worked in an interdisciplinary group of physicians, genetic
    counselors, and basic scientists

##Graduate Research Assistant
Boston College Chestnut Hill, MA
January 2009 - October 2013

-   Utilized existing and wrote novel software to analyze data from
    population scale and family based whole genome and whole exome
    re-sequencing studies to characterize the contribution of rare
    variants

-   Analyzed capture sequencing data from the 1000 Genomes Pilot
    Project

-   Identified a potential causal mutation in a Mendelian form of
    hearing loss

-   Demonstrated the applicability of whole-genome amplified DNA in
    capture sequencing

-   Implemented pedigree aware variant calling algorithm using
    graphical models

##Bioinformatics Programmer 
Cornell University Ithaca, NY
August 2005 - December 2008

-   Implemented bioinformatic data analysis pipelines for DNA
    re-sequencing and genotyping datasets for research projects in
    population and evolutionary genomics

-   Contributed to the publication of over 10 papers in leading
    peer-reviewed journals

##Bioinformatics Programmer 
Stanford University Stanford, CA
February 2001 - July 2002
   
-  Implemented for sequencing and genotyping of candidate genes
    involved in complex diseases

Selected Publications
---------------------

[Variant discovery in targeted resequencing using whole genome amplified
DNA](http://www.ncbi.nlm.nih.gov/pubmed/23837845)

[The functional spectrum of low-frequency coding
variation](http://www.ncbi.nlm.nih.gov/pubmed/21917140)

[A map of human genome variation from population-scale
sequencing](http://www.ncbi.nlm.nih.gov/pubmed/20981092)

[Assessing the evolutionary impact of amino acid mutations in the human
genome](http://www.ncbi.nlm.nih.gov/pubmed/18516229)

[Proportionally more deleterious genetic variation in European than in
African populations](http://www.ncbi.nlm.nih.gov/pubmed/18288194)

[Analysis of concordance of different haplotype block partitioning
algorithms](http://www.ncbi.nlm.nih.gov/pubmed/16356172)

[Full list of publications listed
here](http://orcid.org/0000-0002-3322-4735)

Skills
------

Bioinformatics : GATK, MOSAIK, Freebayes, PLINK, samtools, Picard, Illumina Basespace

Programming : Python, R (Shiny, tidyverse), Perl, bash shell

Databases : MongoDB, MySQL

IT/Computer : AWS EC2, S3, Docker, git

Molecular Biology : PCR, gel electrophoresis

Education
---------

2009-2013
:   *PhD in Bioinformatics at Boston College* Chestnut Hill,
    Massachusetts

Title: Discovering rare variants from populations to families.
Advisor: Gabor T. Marth

2003-2005
:   *MS in Bioinformatics at Medical College of Wisconsin and Marquette
    University* Milwaukee, Wisconsin

Title: Analysis of concordance of haplotype block partitioning
algorithms. Advisor: Michael Olivier

1996-2000
:   *BS in Molecular and Cellular Biology, minor Computer Science
    University of Arizona* Tucson, Arizona

Selected as Outstanding Graduating Senior in Department of Molecular
and Cellular Biology


