Amit R. Indap
=========================

----

>  Specialties: Bioinformatics, Human Genetics, Python


----

Experience
--------------------

Since December 2013 
:   *Research Fellow for [Reproductive Endocrine Unit, Massacusetts General Hospital](http://www.massgeneral.org/reproductiveendocrine//)*
    (Boston, Massachusetts).

    Investigating the genetic basis of reproductive endocrine disorders using whole exome and genome sequencing

January 2009 - October 2013
:   *PhD in Bioinformatics with
    [Gabor T. Marth](http://bioinformatics.bc.edu/marthlab/wiki/index.php/Main_Page)* (Boston College Chestnut Hill, Massachusetts).

    Utilized existing and wrote novel software to analye data from population scale and family based whole genome and whole exome re-sequencing studies to characterize the contribution of rare variants to human genetic diversity and disease risk. 

August 2005 - December 2008 
:   *Bioinformatics Programmer [Cornell University](http://www.cornell.edu/)*
    (Ithaca, NY).

    Implemented bioinformatic data analysis pipelines for DNA re-sequencing and genotyping datasets for research projects in population and evolutionary genomics which helped facilitate the publication of over 10 papers in leading scientific journals.

February 2001 - July 2002 
:   *Bioinformatics Programmer at [ Stanford University ](http://www.stanford.edu)* (Stanford,
    CA).

    Implemented informatics pipeline for sequencing and genotyping of candidate genes involved in complex diseases.

Selected Publications 
----------------------------------

[Variant discovery in targeted resequencing using whole genome amplified DNA](http://www.ncbi.nlm.nih.gov/pubmed/23837845)



[The functional spectrum of low-frequency coding variation](http://www.ncbi.nlm.nih.gov/pubmed/21917140)



[A map of human genome variation from population-scale sequencing](http://www.ncbi.nlm.nih.gov/pubmed/20981092)

   


[Assessing the evolutionary impact of amino acid mutations in the human genome](http://www.ncbi.nlm.nih.gov/pubmed/18516229)



[Proportionally more deleterious genetic variation in European than in African populations](http://www.ncbi.nlm.nih.gov/pubmed/18288194)



[Analysis of concordance of different haplotype block partitioning algorithms](http://www.ncbi.nlm.nih.gov/pubmed/16356172)




Education
---------

2009-2013
:   *PhD in Bioinformatics at Boston College*
    Chestnut Hill , Massachusetts.

    Title: Discovering rare variants from populations to families. Advisor: [Gabor T. Marth
    ](http://bioinformatics.bc.edu/marthlab/wiki/index.php/Main_Page)
    Defended in October 2013.

2003-2005
:   *MS in Bioinformatics at Medical College of Wisconsin and Marquette University* 
	Milwaukee, Wisconsin.

    Title: Analysis of concordance of haplotype block partitional algorithms. Advisor: Michael Olivier

1996-2000
:   *BS in Molecular and Cellularl Biology, minor Computer Science University of Arizona* 	Tucson, Arizona.

    Selected as Outstanding Graduating Senior in Department of Molecular and Cellular Biology.

Skills
------------------------

Bioinformatics
: GATK, MOSAIK, Freebayes, PLINK, samtools, Picard.

Programming
: Python, R, Perl, familiar with Java and C++, [GitHub repo](https://github.com)

Molecular Biology
: PCR, gel electrophoresis.

----

> <indapa@gmail.com> â€¢ [LinkedIn profile](http://www.linkedin.com/in/aindap)

